### ISSUE TITLE
New lines are ignored in markdown code snippets in 1.0.0-alpha.5

### ISSUE THREAD

**Issue by kasvith** [vuejs/vitepress#1175]
### Describe the bug

In Makrdown code snippets new lines are now being ignored. You need to manually add `<space>` with a newline to render it in the docs

### Reproduction

Create a code snippet which has new lines

### Expected behavior

Should render new lines without additional steps

### System Info

```shell
  System:
    OS: Windows 10 10.0.19044
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz 
    Memory: 6.91 GB / 15.96 GB
  Binaries:
    Node: 16.13.0 - C:\Program Files\nodejs\node.EXE       
    Yarn: 1.22.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 8.1.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 104.0.5112.81
    Edge: Spartan (44.19041.1266.0), Chromium (104.0.1293.54)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    vitepress: ^1.0.0-alpha.5 => 1.0.0-alpha.5
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
Yeah, #1170 will fix this.

**Comment by Charles7c** [comment]
> Yeah, #1170 will fix this.

Can this be fixed as well?
![image](https://user-images.githubusercontent.com/25446948/184811462-f83734b4-02e7-478d-87a0-564df5522bc0.png)

**Comment by brc-dd** [comment]
@Charles7c fixed in #1178
